all-collections/_our-team/meg-gutshall.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: page
+
+first_name: Meg
+last_name: Gutshall
+list_order: H
+position: Open Source Advocate and Welcome Wagoner
+photo: /assets/img/people/meg-gutshall.png
+email: meg@rubyforgood.org
+social:
+  - link: https://www.linkedin.com/in/meghan-gutshall/
+    handle: "/meghan-gutshall"
+    icon: linkedin-in
+  - link: https://twitter.com/meg_gutshall
+    handle: "@meg_gutshall"
+    icon: twitter
+  - link: https://github.com/meg-gutshall
+    handle: "/meg-gutshall"
+    icon: github
+  - link: https://www.twitch.tv/meg_gutshall
+    handle: "@meg_gutshall"
+    icon: twitch
+
+write_up:
+  - paragraph: "Meg Gutshall, pronouns she, her, hers, serves in several capacities throughout the Ruby for Good organization. She fell in love with the community at her first Ruby for Good event in 2019 and has been involved ever since, promoting the nonprofit online and at various events as well as contributing code in various project repositories including Coral Gardeners, PartnerBase, and Human Essentials. Her latest task involves her organization skills and knowledge of Jamstack and GitHub to lead the Ruby for Good homepage redesign. You'll immediately recognize her at any event by her bright smile and friendly, outgoing personality. Any questions? She's your girl! If she doesn't know—she'll find out!"
+  - paragraph: "Meg walked a bit of a winding road to get where she is today. She graduated from Temple University with a B.A. in Spanish and worked a few administrative positions before taking a chance on herself and enrolling in a self-paced coding bootcamp. After 3 long years of studying and working three jobs, she's finally finished the curriculum! She currently works part time as a Ruby on Rails consultant and will soon be searching for her first full time developer position."
```

Comment:
Meg walked a bit of a winding road to arrive where she is today. She graduated from Temple University with a B.A. in Spanish and worked in a few administrative positions before taking a chance on herself and enrolling in a self-paced coding bootcamp and hasn't looked back. Meg loves coding, solving both technical and non-technical problems, and building and inclusive technical community.

Reply:
How about this?

> Meg walked a bit of a winding road to arrive where she is today. She graduated from Temple University with a B.A. in Spanish and worked in a few administrative positions before taking a chance on herself and enrolling in a self-paced coding bootcamp. She hasn't looked back since! Meg excels at problem solving (both technical and non-technical) and building an inclusive open-source community.

Reply:
I like that.